### PR TITLE
Deps/upgrade to richie 2.23.0

### PR DESCRIPTION
**Pull request by jbpenrath**
# Purpose

Upgrade all sites to Richie 2.23.0

### CODE REVIEW

File: sites/ademe/src/backend/locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -7,7 +7,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2023-01-16 12:11+0100\n"
+"POT-Creation-Date: 2023-08-08 21:04+0200\n"
```
Comment (sampaccoud) [inline sites/ademe/src/backend/locale/fr/LC_MESSAGES/django.po:10]: some missing translations and some relics in this file?

File: sites/demo/requirements/base.txt [hunk 2]
```diff
@@ -1,10 +1,13 @@
-boto3==1.21.4
-django-check-seo==0.5.1
-django-configurations==2.3.2
+boto3==1.28.9
+django-check-seo==0.6.2
```
Comment (sampaccoud) [inline sites/demo/requirements/base.txt:2]: why not 0.6.4?

File: sites/funmooc/requirements/base.txt [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
-dockerflow==2022.1.0
+dockerflow==2022.8.0
 factory-boy==3.2.1
 gunicorn==20.1.0
-psycopg2-binary==2.9.3
-richie==2.22.0
-sentry-sdk==1.14.0
+psycopg2-binary==2.9.6
+richie==2.23.0
+unidecode==1.3.6 # required by django-check-seo
```
Comment (sampaccoud) [inline sites/funmooc/requirements/base.txt:12]: not required if you upgrade to django-check-seo==0.6.4

**Review by sampaccoud — APPROVED** [review]

File: sites/demo/requirements/base.txt (hunk 2 above)
Comment (jbpenrath) [reply sites/demo/requirements/base.txt:2]: Indeed! Nice catch

**Pull request merged**